packages/react-components/src/components/xy-output-overview-component.tsx

```diff
@@ -9,16 +9,15 @@ import { Experiment } from 'tsp-typescript-client';
 import { TraceOverviewSelectionDialogComponent } from './trace-overview-selection-dialog-component';
 
 const COLOR = {
-    SELECTION_RANGE: '#259fd8',
-    VIEW_RANGE: '#cccccc'
+    SELECTION_RANGE: '#259fd8'
```

Comment:
why was VIEW_RANGE removed? By the way, the coloring in the overview view is broken. Is it related with this change?

Reply:
It was meant to make the "analysis running" message more easily distinguishable from the background, and to use a color that changes according to the Theia theme instead of a harcoded value.

But yes, it makes the shaded areas used when selecting with the right button of the mouse disappear. So I reverted this change to use the hardcoded value.